.github/workflows/ci-tests.yml

```diff
@@ -62,12 +62,15 @@ jobs:
         run: pre-commit run isort --all-files
 
       - name: Black
+        if: always()
```

Comment:
this command is new to me, looks helpful!

wondering tho, shouldn't this be `${{ always() }}` ?

Reply:
It magically works without the brackets :)

Reply:
See https://github.com/dedupeio/dedupe/actions/runs/3261890751/workflow for an example :)